packages/myst-cli/src/project/fromPath.ts

```diff
@@ -136,36 +138,51 @@ function indexFileFromPages(pages: (LocalProjectFolder | LocalProjectPage)[], pa
 /**
  * Build project structure from local file/folder structure.
  */
-export function projectFromPath(
+export async function projectFromPath(
```

Comment:
Gotta make sure we update this with `await` downstream...!

Reply:
Added in curvenote here:
https://github.com/curvenote/curvenote/pull/490